src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/range.tsx

```diff
@@ -19,19 +19,26 @@
 
 import React from 'react';
 import { EuiFormRow, EuiRange } from '@elastic/eui';
-import { VisOptionsSetValue } from 'ui/vis/editors/default';
 
-interface RangeOptionProps {
+interface RangeOptionProps<ParamName extends string> {
   label: string;
   max: number;
   min: number;
-  paramName: string;
+  paramName: ParamName;
   step?: number;
   value: string | number;
```

Comment:
Since it is stored as number
```suggestion
  value: number | '';
```